src/components/UserCard/index.js

```diff
@@ -7,6 +7,7 @@ const UserCard = ({ user }) => {
   useEffect(() => {
     PhotosService.getPhotos(user.id)
       .then((response) => {
+        console.log(response, "Evo ga!")
```

Comment:
@LukaNikcevic77 can you please remove these dummy console logs? Be careful about committing this type of code